.azure-pipelines/azure-pipelines.yml

```diff
@@ -62,7 +62,7 @@ jobs:
   - bash: |
       set -e
       mkdir .maven
-      curl -L "https://downloads.apache.org/maven/maven-3/3.8.1/binaries/apache-maven-3.8.1-bin.tar.gz" -o .maven/maven.tar.gz
+      curl -L "https://archive.apache.org/dist/maven/maven-3/3.8.3/binaries/apache-maven-3.8.3-bin.tar.gz" -o .maven/maven.tar.gz
```

Comment:
Any reason why we you switch to archive? This seems to work too:

https://downloads.apache.org/maven/maven-3/3.8.3/binaries/apache-maven-3.8.3-bin.tar.gz

Reply:
@marchof see description of this PR 😉 that describes why I think that https://archive.apache.org/dist/maven/maven-3/ is more future-proof - this location contains versions that disappear from https://downloads.apache.org/maven/maven-3/

Reply:
🤦‍♂️